public/api-merged.json

```diff
@@ -15940,7 +15940,7 @@
           },
           "example": [
             {
-              "datasourceUid": "-100",
+              "datasourceUid": "__expr__",
```

Comment:
is this file generated?

Reply:
Yes it was and I regenerated it in my latest commit